tests/test_gabor.cc

```diff
@@ -0,0 +1,67 @@
+#include <gtest/gtest.h>
+#include <filesystem>
+#include <fstream>
+#include "test_gabor.h"
+#include "test_download_data.h"
+
+namespace fs = std::filesystem;
+
+using namespace std;
+
+void assert_gpu_results(const std::string cpu_dir, const std::string gpu_dir){
+    ifstream gpu_in(gpu_dir + "/NyxusFeatures.csv");
+    ifstream cpu_in(cpu_dir + "/NyxusFeatures.csv");
+
+    std::string gpu, cpu;
+
+    int count = 0;
+    while(true) {
+        getline(gpu_in, gpu);
+        getline(cpu_in, cpu);
+        
+        if(gpu == "" || cpu == "") break;
+
+        if(gpu != cpu){
+            cout << "gpu: " << gpu << endl;
+            cout << "cpu: " << cpu << endl;
+            continue;
```

Comment:
Should we compare with some sort of tolerance?

Reply:
These lines are no longer needed. I will remove them. If we change from double to float in GPU code, we will need to compare with a tolerance.